ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -218,3 +218,54 @@ periodics:
           sources:
           - secret:
               name: cluster-secrets-azure-file
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: azure-test-production-resource-provider
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --secret-dir=/usr/local/e2e-secrets/azure
+      - --target=[output:stable:sync]
```

Comment:
This is not really needed; I will see whether we can get rid of it w/o needing any changes in the template in a follow-up.